ta/secure_key_services/src/processing_ec.c

```diff
@@ -1083,7 +1083,14 @@ uint32_t sks2tee_ecdh_param_pub(struct sks_attribute_head *proc_params,
 
 	*pub_size = temp;
 
-	return serialargs_get_ptr(&args, pub_data, temp);
+	/* DER encoding: only UNCOMPRESSED EC is currently supported */
```

Comment:
Can you change to 'only uncompressed der short is currently supported'?

Also check the length to see if is >= 0x80 and error out as well (meaning der long).